docs/src/main/asciidoc/redis.adoc

```diff
@@ -615,21 +615,6 @@ ReactiveRedisClient defaultReactiveRedisClient = ReactiveRedisClient.createClien
 ReactiveRedisClient namedReactiveRedisClient = ReactiveRedisClient.createClient("second");
 ----
 
-
-[[dev-services]]
-=== DevServices (Configuration Free Redis)
-
-Quarkus supports a feature called DevServices that allows you to create various datasources without any config.
-What that means practically, is that if you have docker running and have not configured `quarkus.redis.hosts`,
-Quarkus will automatically start a Redis container when running tests or dev-mode, and automatically configure the connection.
-
-Available properties to customize the Redis DevService.
-include::{generated-dir}/config/quarkus-redis-client-config-group-dev-services-config.adoc[opts=optional, leveloffset=+1]
-
-When running the production version of the application, the Redis connection need to be configured as normal,
-so if you want to include a production database config in your `application.properties` and continue to use DevServices
-we recommend that you use the `%prod.` profile to define your Redis settings.
-
```

Comment:
Thanks for extracting this into a separate. 

This was previously linked in the beginning of the doc using the "dev-services" anchor. Do you mind updating the two `NOTE` to link to the added doc?

Reply:
Sure, I'll do that.

Reply:
@machi1990 Done.